src/pug/mixins/homes/sections/Award.pug

```diff
@@ -21,12 +21,14 @@ mixin Award
           | Community Thanks Award badges to the best projects!
     .row.flex-center
       .w-100
-      .col-sm-6.col-lg-3.pt-5.pt-xl-7(data-zanim-timeline!={delay: 0.0} data-zanim-trigger='scroll')
+      .col-lg-6.pt-5.pt-xl-7(data-zanim-timeline!={delay: 0.0} data-zanim-trigger='scroll')
```

Comment:
I like the effect of the two larger cards at the top. 
![screenshot](https://user-images.githubusercontent.com/5799860/236540280-7cbbb03e-b50e-4906-a98b-0fbd38f66850.png)